src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.scss

```diff
@@ -39,4 +47,22 @@
     align-items: center;
     justify-content: center;
   }
+
+  .dataSourceAggregatedViewOuiFlexItem {
+    color: grey;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+    display: inline-block;
+    padding-left: 0%;
```

Comment:
nit: doesnt need `%`

Reply:
sure, I will remove the `%` and push again